3ds2/src/main/java/com/adyen/checkout/adyen3ds2/internal/ui/DefaultAdyen3DS2Delegate.kt

```diff
@@ -78,6 +80,7 @@ internal class DefaultAdyen3DS2Delegate(
     private val coroutineDispatcher: CoroutineDispatcher,
     private val base64Encoder: Base64Encoder,
     private val application: Application,
+    private val analyticsManager: AnalyticsManager?,
```

Comment:
Just double checking: it's okay for this to be nullable, right? Because standalone action components should not do the initial analytics call?

Reply:
I thought that we would only create and initialize analytics from payment components and pass them down to the action components. Also since we can not really create an analytics manager in the action component providers, since we would need the payment method type (for source) which we get only when `handleAction()` is being called, I made them nullable. Unless I missed something and it is possible.

What do you think about this? Probably we should check with product?

Reply:
Oh yes you are right. I didn't think about the technical limitations. 

> I thought that we would only create and initialize analytics from payment components

This is what I also thought, so I think it's actually okay to have them nullable. Let's still double check though